client/src/features/movie-list/component.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import './component.less'
+import MovieItem from '../movie-item'
+import getFilms from'./utils';
+
+class MovieList extends React.Component {
+  
+  constructor(props) {
+    super(props);
+    this.state = {films: null};
+  }
+  
+  componentDidMount() {
+    this.setState(({ films }) => ({ films : getFilms() }));
```

Comment:
`setState` with function-updater is usually used when new state depends on previous one. here you do not have such dependency - please use:
``` js
const films = getFilms();
this.setState({ films  });
```